plugins/api-docs/package.json

```diff
@@ -29,6 +29,7 @@
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.45",
+    "@types/react": "^16.9",
```

Comment:
Got this message, not sure why:

```
$ node scripts/check-type-dependencies.js
Incorrect type dependencies in @backstage/plugin-api-docs:
  Move from devDependencies to dependencies: @types/react
```

Reply:
Yeah you'll have to just obey :P

The reason is, we have a special linter step that checks your module exports. Everything that is exported (visible outside of your module) needs to be an actual dependency, so that the importing package "sees" the right things. Sometimes an interface change is made that directly or indirectly has this effect, and you'll have to move that import up to the dependencies section.

Reply:
(to clarify, I assume you meant that you did this change because of that message - not that you STILL get that message after the change)

Reply:
Yah exactly, I justed added as for the reviewers as it didn't feel obviously